app/src/main/java/com/kickstarter/viewmodels/ActivityFeedViewModel.java

```diff
@@ -61,86 +61,123 @@
 
     /** Emits an activity when project update should be shown. */
     Observable<Activity> goToProjectUpdate();
+
+    /** Emits a SurveyResponse when it should be shown. */
+    Observable<SurveyResponse> goToSurvey();
+
+    /** Emits a boolean indicating whether activities are being fetched from the API. */
+    Observable<Boolean> isFetchingActivities();
+
+    /** Emits a boolean that determines if a logged-out, empty state should be displayed. */
+    Observable<Boolean> loggedOutEmptyStateIsVisible();
+
+    /** Emits a logged-in user with zero activities in order to display an empty state. */
+    Observable<Boolean> loggedInEmptyStateIsVisible();
+
+    /** Emits a list of unanswered surveys to be shown in the user's activity feed */
+    Observable<List<SurveyResponse>> surveys();
   }
 
-  final class ViewModel extends ActivityViewModel<ActivityFeedActivity> implements ActivityFeedAdapter.Delegate,
-    Inputs, Outputs {
+  final class ViewModel extends ActivityViewModel<ActivityFeedActivity> implements
+    ActivityFeedViewModel.Inputs, ActivityFeedViewModel.Outputs {
```

Comment:
on the other hand, these make sense to just be `Inputs` and `Outputs` because they are defined right above and it works the exact same way in every view model.